### PR TITLE
Updated german translations

### PR DESCRIPTION
**Pull request by peat80**
Updated german strings

### CODE REVIEW

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -40,10 +41,12 @@
     <string name="use_safe_stores">Sichere Stores nutzen (Aptoide)</string>
     <string name="root_install">Root-Installation</string>
     <string name="source_apkmirror" translatable="false">ApkMirror</string>
-    <string name="source_fdroid" translatable="false">F-Droid (Main)</string>
-    <string name="source_izzy" translatable="false">F-Droid (Izzy)</string>
+    <string name="source_fdroid" translatable="false">F-Droid Offiziell</string>
+    <string name="source_izzy" translatable="false">F-Droid Izzy</string>
```
Comment (AnonymousWP) [inline app/src/main/res/values-de/strings.xml:45]:
```suggestion
    <string name="source_fdroid" translatable="false">F-Droid (Main)</string>
    <string name="source_izzy" translatable="false">F-Droid (Izzy)</string>
```
Again, you shouldn't modify those.

**Review by AnonymousWP — CHANGES_REQUESTED** [review]
Same as here: https://github.com/rumboalla/apkupdater/pull/467#discussion_r1372041620. Also, do not close this PR like you did previously, but you can apply my suggestions.

File: app/src/main/res/values-de/strings.xml (hunk 1 above)
Comment (peat80) [reply app/src/main/res/values-de/strings.xml:45]: Just look at the main strings.xml in the values folder. 
Those strings were also changed and have the brackets removed.

Comment (AnonymousWP) [reply app/src/main/res/values-de/strings.xml:45]: Yes, that's fair, but you shouldn't put "Officiziell". Keep it as-is with out the brackets.

Comment (rumboalla) [reply app/src/main/res/values-de/strings.xml:45]: Maybe I should mark these as translatable.

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -40,10 +41,12 @@
...
     <string name="source_aptoide" translatable="false">Aptoide</string>
     <string name="source_github" translatable="false">GitHub</string>
+    <string name="source_gitlab" translatable="false">GitLab</string>
+    <string name="source_apkpure" translatable="false">APKPure</string>
```
Comment (rumboalla) [inline app/src/main/res/values-de/strings.xml:49]: There's no need to add strings that are marked as non traslatable

**Pull request merged**